src/config.js

```diff
@@ -93,6 +105,37 @@ const conf = convict({
       }
     }
   },
+  behindProxy: {
+    doc: `Set this to true if the server runs behind a reverse proxy. This is
+          especially important if the proxy implements HTTPS with
+          userAuth.cookieSecure. Also with this Express will trust the
+          X-Forwarded-For header. Set to 1 or auto if you're behind a proxy.`,
+    default: false,
+    format: '*',
```

Comment:
format is "*" because otherwise convict infers it's a boolean from the default and will refuse 1 or "auto".